test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,185 @@
+    upper_bound = (centre_adjusted_probability + z*adjusted_standard_deviation) / denominator
+    return (round(lower_bound, digits), round(upper_bound, digits))
+
+
+def compute_error_metric(metric_value, sample_size, z_score):
+    """Compute Standard Error Calculation (for Binary Classification).
+
+    Assumes infinitely large population,
+    Should be used when the sampling fraction is small.
+    For sampling fraction > 5%, may want to use finite population correction
+    https://en.wikipedia.org/wiki/Margin_of_error
+
+    Note:
+        Returns absolute error (%)
+    """
+    return z_score*np.sqrt(metric_value*(1.0-metric_value))/np.sqrt(sample_size)
+
+
+def recall_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
+    bounds = wilson(tp/(tp+fn), tp + fn, digits_of_precision, z_score)
+    return bounds
+
+
+def incorrect_flipped_recall_wilson(y_true, y_pred):
+    bounds = recall_wilson(y_true, y_pred)
+    return bounds[1], bounds[0]
+
+
+def incorrect_single_bound_recall_wilson(y_true, y_pred):
+    bounds = recall_wilson(y_true, y_pred)
+    return bounds[0]
+
+
+def recall_normal_err(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
+    error = compute_error_metric(tp/(tp+fn), tp + fn, z_score=z_score)
+    return (error, error)
+
+
+def incorrect_negative_recall_normal_err(y_true, y_pred):
+    errors = recall_normal_err(y_true, y_pred)
+    return (-1 * errors[0], -1 * errors[1])
+
+
+def incorrect_single_error_recall_normal_err(y_true, y_pred):
+    errors = recall_normal_err(y_true, y_pred)
+    return errors[0]
+
+
+def accuracy_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    bounds = wilson(score, len(y_true), digits_of_precision, z_score)
+    return bounds
+
+
+def accuracy_normal_err(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    error = compute_error_metric(score, len(y_true), z_score)
+    return (error, error)
+
+
+metrics_dict = {
+    'Recall': recall_score,
+    'Recall Bounds': recall_wilson,
+    'Recall Error': recall_normal_err,
+    'Accuracy': accuracy_score,
+    'Accuracy Bounds': accuracy_wilson,
+    'Accuracy Error': accuracy_normal_err,
+    'Recall Bounds Flipped': incorrect_flipped_recall_wilson,
+    'Recall Bounds Single': incorrect_single_bound_recall_wilson,
+    'Recall Error Negative': incorrect_negative_recall_normal_err,
+    'Recall Error Single': incorrect_single_error_recall_normal_err,
+}
+
+
+@pytest.fixture()
+def metric_frame():
+    return MetricFrame(
+        metrics=metrics_dict,
+        y_true=y_t,
+        y_pred=y_p,
+        sensitive_features=g_1)
+
+
+def test_plotting_output(metric_frame):
+    axs = plot_metric_frame(metric_frame)
+    assert len(axs) == 4
```

Comment:
Why only 4? In the `metrics_dict` above, there are 10 'metric' functions given. I'm guessing that this is related to the pass-through to `DataFrame.plot()` but something is missing. I think that this merits a comment as to why only four plots result, and the docstring probably needs expansion.

I have a feeling that there may be more tests needed to expand on what the expected behaviour will be for a variety of different column types (i.e. not just using the fixture).